src/FakeItEasy/A.cs

```diff
@@ -44,7 +44,7 @@ public static T Fake<T>(Action<IFakeOptions<T>> optionsBuilder)
         {
             Guard.AgainstNull(optionsBuilder, nameof(optionsBuilder));
 
-            return (T)FakeAndDummyManager.CreateFake(typeof(T), options => optionsBuilder((IFakeOptions<T>)options));
+            return (T)FakeAndDummyManager.CreateFake(new DummyCreationSession(), typeof(T), options => optionsBuilder((IFakeOptions<T>)options));

```

Comment:
We'd had a recent discussion about the servicelocator anti-pattern and how we'd be better off just using it at the static entry points, so I strove for that. I'm happy to inject it lower, though!